examples/frameworks/twisted/twistd_app.py

```diff
@@ -48,7 +48,8 @@ def cb_twistd(self, *la):
             IServiceCollection(self.app).stopService()
             self.running = False
         else:
-            sys.exc_clear()
+            if sys.version_info[0] < 3:
```

Comment:
Master does not support python 2 anymore so there's no need to check for this, you can just remove the line.